lib/get-tasks.js

```diff
@@ -1,27 +1,25 @@
 /**
  * Acceptable formats:
- * - #20jt35r
  * - #CORE-123
  */
 const getTasks = (source) => {
-    const ticketRefRegex = /(?<=\s|^)#(([A-Z]{2,10}-\d+)|(\w{7,10}))/g;
-    const ticketIdRegex = /#(?<taskId>([A-Z]{2,10}-\d+)|(\w{7,10}))/;
-    const customTicketIdRegex = /#[A-Z]{2,10}-\d+/;
-    const ids =
-      source.match(ticketRefRegex)?.map(v => ({
-        ...v.match(ticketIdRegex).groups,
-        isCustom: customTicketIdRegex.test(v),
-      })) || [];
+  const taskRefRegex = /(?<=\s|^)#[A-Z]{2,10}-\d+/g;
+  const refs = source.match(taskRefRegex) || [];
```

Comment:
We agreed to put the task ids at the end, so let's also check that to not have chaos :D
To do that and still be able to catch multiple tasks, we need to match the whole task ids string as one and split it:
<img width="614" alt="image" src="https://github.com/user-attachments/assets/9b8d1f34-2890-4e17-9979-cab303868f9d">
```suggestion
  const taskRefRegex = /(\s+#[A-Z]{2,10}-\d+)+\s*$/;
  const allTaskIds = taskRefRegex.match(source)?.[0]?.trim() ?? '';
  const refs = allTaskIds.split(/\s+/);
```

Reply:
At first I've liked the idea and implemented it 94a88d1c2bacb8cfbf5c82c0984e89a414a40dc6. But now I've realized there are some repos where regularly task reference was added at the beginning of the title e.g. [RampNetwork/data-eng-core](https://github.com/RampNetwork/data-eng-core/pulls) or [RampNetwork/ramp-infra](https://github.com/RampNetwork/ramp-infra/pulls).

I am afraid enforcing issue reference at the end of title for all repositories may bring some chaos, what do you think?

Reply:
I don't have access to data-eng-core, but looking at the infra examples I'd say putting the task id in the front is incorrect, as the PR title should also stick to the conventional commits convention with the change type & scope at the beginning. So I actually think that it's good to enforce this.

Reply:
we may extend the error message to include that the task ids must be "at the end" of PR title.

Reply:
Done c6c277eeffdbb6b29f9b87d3d6d332f5ecee5062